tests/utils/torch_utils_test.py

```diff
@@ -53,3 +55,13 @@ def __init__(self, sub_model):
     # Apply our workaround torch_utils.load_state_dict_in and assert True.
     model_2 = torch_utils.load_state_dict_in(state_dict, model_2)
     assert torch_utils.are_models_equal(model, model_2)
+
+
+def test_slicer():
+    x = torch.randn(10, 2, 16384)
```

Comment:
Hmm, this doesn't need to be sliced no? 
So it actually doesn't unit test, or I'm missing something? 
Can we test with other shapes

Reply:
No you are right